Packages/Core/Runtime/Variables/AtomVariable.cs

```diff
@@ -142,5 +156,23 @@ public IObservable<T> ObserveChange()
             );
         }
         #endregion // Observable
+
+        private T RunPreChangeTransformers(T value)
+        {
+            var preProcessedValue = value;
+            if (PreChangeTransformers != null && PreChangeTransformers.Count > 0)
+            {
+                for (var i = 0; i < PreChangeTransformers.Count; ++i)
+                {
+                    var Transformer = PreChangeTransformers[i];
+                    if (Transformer != null)
```

Comment:
filter all "null"s out of the List in OnEnable() to reduce null checks against UnityEngine.Objects (which ScriptableObjects are)?

Reply:
Don't think we should skip the null check since it is possible to add change transformers at runtime (you could potentially add null).